### PR TITLE
Preset Drupal

### PR DESCRIPTION
**Pull request by killua99**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| Fixed tickets | #118

Adding Preset for Drupal 8

### CODE REVIEW

File: src/Application/Adapters/Drupal8/Preset.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace NunoMaduro\PhpInsights\Application\Adapters\Drupal8;
```
Comment (nunomaduro) [inline src/Application/Adapters/Drupal8/Preset.php]: Can we rename this to `Drupal` only, and maybe only maintain the latest version?

Comment (killua99) [reply src/Application/Adapters/Drupal8/Preset.php]: 🤔 Sure thing, I'm thinking Drupal 9 will be release just in 1y, I guess it doesn't really matter, perhaps we could create a Drupal9 later on, idk. I'll put Drupal for now

Comment (nunomaduro) [reply src/Application/Adapters/Drupal8/Preset.php]: Thank you so much!

Comment (killua99) [reply src/Application/Adapters/Drupal8/Preset.php]: Let me know what else shall I add. I didn't edit any readme file.

**Pull request merged**